client/document.go

```diff
@@ -83,12 +89,9 @@ func newEmptyDoc() *Document {
 }
 
 // NewDocFromMap creates a new Document from a data map.
-func NewDocFromMap(data map[string]any) (*Document, error) {
+func NewDocFromMap(data map[string]any, sd SchemaDescription) (*Document, error) {
```

Comment:
todo: Please review the `SchemaVersionID` property on `Document` (if you haven't done so already, if so please document why it is not set), it looks like it should/can be set in these functions taking a SchemaDescription.

Reply:
The property is actually never used. I'll remove it.

Reply:
✅